scss/bitstyles/ui/section-heading.stories.mdx

```diff
@@ -55,87 +56,32 @@ A heading with optional actions, badges and labels. Be sure to use the correct h
   </ul>
 </details>
 
-## Section heading
+### With text only
 
 <Canvas>
-  <Story name="Section heading">
-    {`
-      <div class="u-padding-m-bottom u-border-grayscale-light-bottom">
-        <h3 class="u-margin-0 u-margin-m-right">Recent bookings</h3>
-      </div>
-    `}
-  </Story>
+  <Story id="ui-content-section-heading--text-only" />
 </Canvas>
 
-## Section headings with labels and badges
-
-Next to the heading text is a good place to add important information about the content referred to, such as its current state or counts.
```

Comment:
The mdx files are meant to provide some documentation for human readers — developers who might come to use this component, or need a layout like this but aren’t sure how best to create it. So maybe this line of text could be updated/expanded instead of removed?